_data/payments.yml

```
@@ -203,6 +203,13 @@ websites:
       - totp
     doc: https://help.tebex.io/en/articles/2319372
 
+  - name: Todo Pago
+    url: https://portal.todopago.com.ar/
```

This seems to be the main page.
```suggestion
    url: https://todopago.com.ar/
```